floss/identify.py

```diff
@@ -151,8 +152,8 @@ def find_decoding_function_features(vw, functions, disable_progress=False) -> Tu
 
             f = viv_utils.Function(vw, function_address)
 
-            function_data = {"meta": get_function_meta(f), "features": list()}
-
+            # function_data = {"meta": get_function_meta(f), "features": [] }
```

Comment:
please remove commented out code. we can use git history to resurrect it if necessary.